api/src/main/java/org/openmrs/module/queue/model/Queue.java

```diff
@@ -62,30 +63,9 @@ public class Queue extends BaseChangeableOpenmrsMetadata {
 	@JoinColumn(name = "status_concept_set", referencedColumnName = "concept_id")
 	private Concept statusConceptSet;
 	
-	@OneToMany(mappedBy = "queue", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
-	private List<QueueEntry> queueEntries;
```

Comment:
I'm a little confused by this part of the change. Does this mean that queues no longer have queue entries?

Reply:
@ibacher yes, basically.  It says that "Queue Entries" belong to a Queue, but a Queue does not directly have access to the entries that are associated with it.  Can you find another example of where it's been our practice to have `data` belong to `metadata` directly?  You can't get a `List<PatientProgram>` from a `Program` for example, even if one might think they want to do something like `program.getEnrollments()`

Reply:
Nope. It always bothered me that we had this direct linkage! In retrospect, I should've just read the ticket.

Reply:
The refactoring makes sense. Thanks @mseaton